library/src/scala/quoted/Quotes.scala

```diff
@@ -464,6 +464,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     /** Methods of the module object `val ClassDef` */
     trait ClassDefModule { this: ClassDef.type =>
+      @experimental def apply(cls: Symbol, parents: List[Tree /* Term | TypeTree */], body: List[Statement]): ClassDef
```

Comment:
There's no example where we pass an actual term (e.g. `new Foo("a")` to pass a parameter to the parent), is that actually supported?

Reply:
Added `tests/run-macros/newClassExtendsClassParams` to test this feature.

It actually interacts butefully with quoted expressions. We can define the parents as
```scala
  val parents = List('{ new Foo(1) }.asTerm)
```